test/plugin/scenarios/webflux-scenario/bin/startup.sh

```diff
@@ -18,7 +18,4 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -jar ${agent_opts} "-Dskywalking.agent.service_name=webflux-projectA-scenario" ${home}/../libs/webflux-projectA-scenario.jar &
-sleep 1
-
-java -jar ${agent_opts} "-Dskywalking.agent.service_name=webflux-projectB-scenario" ${home}/../libs/webflux-projectB-scenario.jar &
+java -jar ${agent_opts} ${home}/../libs/webflux-scenario.jar &
```

Comment:
You just move two package into one, right?

Reply:
Yes, make request test in one